modules/learn/pages/services-and-indexes/indexes/storage-modes.adoc

```diff
@@ -39,6 +33,12 @@ To resume indexing operations on a node where the Indexer has paused due to low
 +
 Note that attempting to delete bucket-data _selectively_ during an out-of-memory condition does not succeed in decreasing memory-usage; since without memory, such requested deletions cannot themselves be processed.
 
+In cases where recovery requires an Index-Service node to be restarted, the node's resident memory-optimized indexes are rebuilt from the snapshots retained on disk.
+Following the node's restart, these indexes remain in the `BUILDING` state until all information has been read into memory: then, final updates are made with the indexes in `ONLINE` state.
```

Comment:
The correct state is "Warmup" instead of "Building"

Reply:
Done